live/global.ini

```
@@ -42165,36 +42165,36 @@ bountymarker_lost=%02i:%02i Seit dem letzten Aufruf von %ls
 cargoTransfer_HUD_AwaitingArrival=Warten auf Ihre Ankunft um
 cargoTransfer_HUD_BeginTransfer=um den Transfer zu beginnen.
 cargoTransfer_HUD_CargoSlot=Frachtdeck
-cargoTransfer_HUD_CargoSlotObstructed=Behinderung der Ladefläche verhindert den Transfer.
+cargoTransfer_HUD_CargoSlotObstructed=Blockierung des Ladebereich, die den Transfer verhindert.
```

```suggestion
cargoTransfer_HUD_CargoSlotObstructed=Der Ladebereich ist blockiert und verhindert den Transfer.
```

Oder sowas. Das aktuelle ist irgendwie komisch formuliert